src/nvim/memline.c

```diff
@@ -619,18 +616,10 @@ void ml_timestamp(buf_T *buf)
   ml_upd_block0(buf, UB_FNAME);
 }
 
-/*
- * Return FAIL when the ID of "b0p" is wrong.
- */
-static int ml_check_b0_id(ZERO_BL *b0p)
+/// Checks whether the ID of "b0p" is correct.
```

Comment:
Perhaps `whether blabla is valid, i.e. the ID indicates the swap file is in a known format.`